lib/console_client/console_io.rb

```diff
@@ -22,21 +22,15 @@ def exit
       @device.exit
     end
 
-    def exit?
-      @exit
-    end
-
-    def exit=(exit)
-      @exit = exit
-    end
-
     def get_input(valid_input, error_message = '', prompt = '')
-      begin
+      input = nil
```

Comment:
This feels like it's doing more than just get input. It's validating/displaying errors as well...Hmm.

Reply:
Would it be better named as something like `get_validated_input`? Or is the function doing too much? I figured that this set of actions is repeated several times in various places, so needs to be encapsulated in a method somewhere.